Makefile

```diff
@@ -25,6 +25,8 @@ GOBIN := $(shell $(GO) env GOBIN)
 ifeq ($(GOBIN),)
 GOBIN := $(FIRST_GOPATH)/bin
 endif
+CGO_ENABLED ?= 1
+export CGO_ENABLED
```

Comment:
How does this relate to the subject of the PR?

Reply:
It is a mistake.  Removing.